robert_beverage/scripts/web_server

```diff
@@ -1,23 +1,33 @@
 #!/usr/bin/env python
+import os
 import rospy
 import socket
 import SocketServer
 import SimpleHTTPServer
+from urlparse import urlparse
+import rospkg
 
 rospy.init_node('web_server_node')
 rospy.loginfo('Inside web_server script')
+rospack = rospkg.RosPack()
 
 HOST = socket.gethostname()
 PORT = 8080
 address = ("", PORT)
+os.chdir('/')
 
 class RequestHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
     def do_GET(self):
-        if self.path.startswith("/beverage?"):
-            info = self.path.strip("/")[9:]
-            coor = dict(x.split("=")for x in info.split("&"))
-            rospy.loginfo("x=%s\ty=%s", coor['x'], coor['y'])
-        self.path = "/workspace/robert/base/src/robert/robert_beverage/scripts/index.html"
+        try:
+            if self.path.startswith("/beverage?"):
+                query = urlparse(self.path).query
+                coor = dict(qc.split("=")for qc in query.split("&"))
```

Comment:
please try this or other answers from this thread https://stackoverflow.com/a/23615281